gripper_action_controller/CHANGELOG.rst

```diff
@@ -0,0 +1,122 @@
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
```

Comment:
Should I remove the changelog .?

Reply:
yes please
and while you are at it, please also set the version of this package to the same number as the rest on master, that saves me an iteration of fixing things when releasing :)